src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderConfig.tsx

```diff
@@ -179,59 +140,58 @@ export const LimitOrderConfig = ({
     setLimitPriceDirection(newPriceDirection)
   }, [priceDirection, setLimitPriceDirection])
 
-  const handlePriceChange = useCallback(() => {
-    // onChange will send us the formatted value
-    // To get around this we need to get the value from the onChange using a ref
-    // Now when the preset price mode buttons are clicked the onChange will not fire
```

Comment:
Keep me?

Reply:
Done in [e905124](https://github.com/shapeshift/web/pull/8970/commits/e9051249766cda9ac47762f1ce453ba3f1e35c9f)